testsuite/E40.py

```diff
@@ -32,6 +32,13 @@
     print('imported foo')
 finally:
     print('made attempt to import foo')
+#: Okay
+if False:
+    import foo
+#: Okay
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", DeprecationWarning)
+    import foo
 
 import bar
```

Comment:
oh actually in order to demonstrate this test you need to have an `import bar` after each of the blocks that would have triggered E402

Basically need to add the `import bar` back to the test above and add `import bar` to the `if` test

Reply:
Er, I think I fixed it?

Reply:
awesome, looks great